Resources/Prototypes/Roles/Jobs/Command/captain.yml

```diff
@@ -10,15 +10,26 @@
   access:
   # All of em.
   # Could probably do with some kind of wildcard or whatever to automate this.
+  # HELL FUCKING YEAH WE COULD
```

Comment:
Thank you for volunteering

Reply:
I was going to do this in this PR but it was non-trivial for job prototypes. I would imagine I can't just use `IPrototypeManager` to find all the existing access prototypes because they might not be loaded yet.